### PR TITLE
add possible linkedin phishing site

### PR DESCRIPTION
**Pull request by allaway**
Stumbled upon this by accident. Redirects to an unknown, possibly malicious site (http://www6.linkedinin.com).

## Domain/URL/IP(s) where you have found the Phishing:
linkedinin.com
www6.linkedinin.com

## Related external source
n/a, discovered myself by accident...

## Describe the issue
This typo is a valid domain that redirects to a blank website that does not appear to be affiliated w/linkedin. 

### Screenshot
<--! If you feel a screenshot can say more than 1000 hard drives, do please feel free to add it here :smiley:

**TIP**: Place your mouse on the line just above the `</details>` 
and paste your screenshot and make sure that there is at least one
line spacing before and after the image code line. The tip will add"
one line after the paste :wink: -->

<details><summary>Click to expand</summary>

</details>

### CODE REVIEW

**Review by spirillen — CHANGES_REQUESTED** [review]
Please add this one to the https://github.com/mitchellkrogza/phishing/blob/main/add-wildcard-domain as `linkedinin.com` that will protect against any  subdomain renaming

File: add-domain [hunk 1]
```diff
@@ -698,6 +698,7 @@ l9e5372b.justinstalledpanel.com
 la95a1ab.justinstalledpanel.com
 lgin.msa.trafficmanager.net
 licencefeedback.com
+linkedinin.com
```
Comment (allaway) [inline add-domain:701]:
```suggestion
```

File: add-domain [hunk 2]
```diff
@@ -1016,6 +1017,7 @@ www.web-lgviolationformcenter.com
 www.xserhats.ml
 www.youthful-engelbart.34-125-197-109.plesk.page
 wwwgosuslugj.ru
+www6.linkedinin.com
```
Comment (allaway) [inline add-domain:1020]:
```suggestion
```

**Comment by allaway** [comment]
@mitchellkrogza , thanks for the quick review! Hopefully I followed your request correctly.

**Comment by allaway** [comment]
@spirillen thanks for the quick review! Hopefully I followed your request correctly.

**Review by spirillen — APPROVED** [review]

**Comment by spirillen** [comment]
> @spirillen thanks for the quick review! Hopefully I followed your request correctly.

Seems good, just had a few days off :+1:

**Pull request merged**